login.py

```
@@ -2,9 +2,9 @@
 import sqlite3
 
 def login(username, password):
```

Ruff (0.14.6)

Unused function argument: `username`

(ARG001)